sycl/test-e2e/format.py

```diff
@@ -173,6 +174,12 @@ def execute(self, test, litConfig):
                 (backend, _) = sycl_device.split(":")
                 triples.add(get_triple(test, backend))
 
+            if (
+                "build-and-run-mode" in test.requires
+                and test.config.fallback_build_run_only
+            ):
+                ignore_line_filtering = True
```

Comment:
Why cannot that be done at the place when `ignore_line_filtering` will be used?

Reply:
Ok, i put this right outside of the loop where `ignore_line_filtering` is used.